lithium/lithium.py

```diff
@@ -207,10 +214,12 @@ def addArgs(self, parser):
         grp_add.add_argument(
             "--chunkstart", type=int,
             default=0,
-            help="For the first round only, start n chars/lines into the file. Best for max to divide n. [Mostly intended for internal use]")
+            help="For the first round only, start n chars/lines into the file. Best for max to divide n. "
+            "[Mostly intended for internal use]")
```

Comment:
Ditto re: below thread on

    help="..."
    "..."

vs.

    help="..."
         "..."

Not saying it *needs* to change, just that it should be consistent.